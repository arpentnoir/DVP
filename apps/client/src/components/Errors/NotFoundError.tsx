import { Text } from '@dvp/vc-ui';
import { Stack } from '@mui/material';

export const NotFoundError = () => {
  return (
    <Stack
      sx={{
        flex: 1,
        justifyContent: 'center',
        alignItems: 'center',
        display: 'flex',
      }}
    >
      <Stack spacing={2}>
        <Text variant="h6" fontWeight="bold">
          404
        </Text>
        <Text variant="h4">PAGE NOT FOUND</Text>
        <Text>If you entered a web address please check it was correct.</Text>
      </Stack>
    </Stack>
  );
};
