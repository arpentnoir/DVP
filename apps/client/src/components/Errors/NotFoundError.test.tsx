import { render } from '@testing-library/react';
import { NotFoundError } from './NotFoundError';

describe('NotFoundError', () => {
  it('should render correctly', () => {
    const { baseElement } = render(<NotFoundError />);

    expect(baseElement).toMatchSnapshot();
  });
});
