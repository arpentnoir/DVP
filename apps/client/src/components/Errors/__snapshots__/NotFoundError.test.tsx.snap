// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundError should render correctly 1`] = `
<body>
  <div>
    <div
      class="css-1ootpi0-MuiStack-root"
    >
      <div
        class="css-1p5q5e5-MuiStack-root"
      >
        <h6
          class="MuiTypography-root MuiTypography-h6 css-tjvwag-MuiTypography-root"
          tabindex="0"
        >
          404
        </h6>
        <h4
          class="MuiTypography-root MuiTypography-h4 css-5lbw0b-MuiTypography-root"
          tabindex="0"
        >
          PAGE NOT FOUND
        </h4>
        <p
          class="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
          tabindex="0"
        >
          If you entered a web address please check it was correct.
        </p>
      </div>
    </div>
  </div>
</body>
`;
